core/src/main/scala/com/microsoft/azure/synapse/ml/logging/Usage/FabricTokenServiceClient.scala

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.logging.Usage
+
+import java.util.UUID
+import java.net.URL
+import java.net.InetAddress
+import java.lang.management.ManagementFactory
+import spray.json.DefaultJsonProtocol.StringJsonFormat
+import com.microsoft.azure.synapse.ml.logging.Usage.FabricConstants._
+import spray.json.{JsArray, JsObject, JsValue, _}
+import com.microsoft.azure.synapse.ml.logging.common.WebUtils.usageGet
+import com.microsoft.azure.synapse.ml.logging.SynapseMLLogging
+
+class FabricTokenServiceClient {
+  val resourceMapping = Map(
+    "https://storage.azure.com" -> "storage",
+    "storage" -> "storage",
```

Comment:
seems odd that you have keyname -> keyname values in this map

Reply:
I felt so too, but this part align to what Cruise's has done on the Python client. For now, let's go with this. If things change, I will update it here.